lua/terrortown/menus/gamemode/equipment/base_equipment.lua

```diff
@@ -186,7 +186,16 @@ function CLGAMEMODESUBMENU:Populate(parent)
     end
 
     -- now add custom equipment settings
-    equipment:AddToSettingsMenu(parent)
+    if isfunction(equipment.AddToSettingsMenu) then
+        equipment:AddToSettingsMenu(parent)
+    else
+        Dev(
+            1,
+            "Weapon '"
+                .. equipment:GetClass()
+                .. "' does't use the weapon_tttbase and can therefore not populate the settings panel."
```

Comment:
```suggestion
                .. "' doesn't use the weapon_tttbase and cannot be added to the settings panel."
```

Reply:
ah shit, I merged it but your suggestion was wrong. It is added to the panel, only custom settings can't populate. I should probably revert it?